### PR TITLE
Added "Request access" template

### PR DESCRIPTION
**Pull request by oliviaflory**
Added a "request access" template to the design kit repo for adopters outside of DGC who would like early access to the design kits. 

The templates asks a few questions to help us better assess if we should give them access to the design kits.

### CODE REVIEW

**Comment by oliviaflory** [comment]
Linda and Wonil, please let me know what you think about the issue template I created for adopters to request access to our design kits. You can see the template in the "Files changed" tab at the top of this PR.

File: .github/ISSUE_TEMPLATE/request-access.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+about: Request access to the IBM.com Expressive and IBM.com Library design kits
+title: Design kit access request
+labels: adopter support
+assignees: ljcarot, wonilsuhibm, oliviaflory
+
+---
+
+Thanks for your interest in the IBM.com Design kits!
+
+We are in a closed adoption phase at the moment, please answer the following questions to help us better understand your project and someone from the team will reach out to you soon.
```
Comment (ljcarot) [inline .github/ISSUE_TEMPLATE/request-access.md]: We are in a closed adoption phase at the moment.  Please answer the following questions to help us better understand your interest.  Someone from DDS will reach out to you soon regarding your request for access.

Comment (oliviaflory) [reply .github/ISSUE_TEMPLATE/request-access.md]: Since we are not DDS anymore, I replaced DDS with Carbon for IBM.com

Comment (oliviaflory) [reply .github/ISSUE_TEMPLATE/request-access.md]: This has been updated

**Comment by wonilsuhibm** [comment]
All look good. One small suggestion: maybe an `Additional information` section at the bottom?

**Comment by oliviaflory** [comment]
@wonilsuhibm I added: "Any additional information you'd like to share about your project" at the bottom of the template

**Pull request merged**